mkinit/static_mkinit.py

```diff
@@ -83,6 +83,7 @@ def autogen_init(
     logger.info(
         "Autogenerating __init__ for modpath_or_name={}".format(modpath_or_name)
     )
+    options = _ensure_options(options)

```

Comment:
This `_ensure_options` ends up getting called here now in addition to in `formatting._initstr`, but I didn't see any obvious ways to refactor